static/header.less

```diff
@@ -9,7 +9,7 @@
 }
 .header {
     display: flex;
-    align-items: center;
+    align-items: stretch;
```

Comment:
Not sure what `center` was doing here. `stretch` will actually vertically center everything.